### ISSUE TITLE
Airplay does not find any AppleTV

### ISSUE THREAD

**Issue by ph00lt0** [futo-org/grayjay-android#1255]
### What happened?

According to the readme the app should have airplay support. I see alsof issues reported on LG TVs but i wonder does the app even have working airplay support? It doesnt seem to find any of my Apple TV 4K. 

### Grayjay Version

249

### What plugins are you seeing the problem on?

All

### Plugin Version

N/A

### When do you experience the issue?

- [ ] While logged in
- [ ] While logged out
- [X] N/A

### Relevant log output

```shell
N/A
```

**Comment by GenericUsernameIsNotAvailable** [comment]
I've turned casting off as even when it manages to find the tv, it fails to load or make any contact with the TV

**Comment by codebymatter** [comment]
Airplay doesn't really work all that well. My pqrents TV has Airplay 2.0 features but it's not an Apple product. Not sure if anyone has actually managed to cast through AirPlay.

**Comment by FaultlineTM** [comment]
Yes encountering same behavior as mentioned above for ph00lt0. For context, I have the grayjay apple desktop app (the arm version) for an m1 mac laptop. It does show my apple tv (the apple tv 4k 3rd gen) as something I can cast to. Unfortunately though, upon selecting that option it wont actually cast properly.

**Comment by bjrnblm** [comment]
I have the same issue, it has never worked for me. For all shown devices (AppleTv, MacBook, Sonos) it just keeps spnning without something happening.

![Image](https://github.com/user-attachments/assets/371b4367-f45b-4f1d-9333-83133672f328)

![Image](https://github.com/user-attachments/assets/f7c6ca9f-0b4b-4e3b-994e-41bc500c2402)

**Comment by Zvonimir-FUTO** [comment]
Closing this as not planned since support for Airplay has been dropped.
